minikube/startup.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-set -euo pipefail # exit on any failures
+set -euxo pipefail # exit on any failures
```

Comment:
Is there a reason for keeping the tracing here?

Reply:
Nope, it's an accidental commit. I've removed it. :+1: